04/app.js

```diff
@@ -0,0 +1,35 @@
+// ************
+// TASK 4.01 **
+// ************
+
+const divEl = document.querySelectorAll('div');
+
+// let timeout = divEl.forEach(e => {
+//   temp = e.dataset.time;
+//   console.log(temp);
+// });
+
+divEl.forEach(e => {
+  e.addEventListener('click', el => {
+    setTimeout(function() {
+      // el.preventDefault(); // or maybe stopPropagation?
```

Comment:
ani jedno, ani drugie

Jak klikasz w zielony box to wszystkie 3 powinny się pokolorować w odpowiednim odstępie czasu.